### PR TITLE
Fix issue #443

### PR DESCRIPTION
**Pull request by justnoone00**

### CODE REVIEW

File: activities/Fototoon.activity/lib/tutorial.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+define(["webL10n"], function (l10n) {
+//define([], function () {
```
Comment (llaske) [inline activities/Fototoon.activity/lib/tutorial.js]: Remove commented lines

File: activities/Fototoon.activity/lib/tutorial.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	var tutorial = {};
+
+	tutorial.start = function(language) {
+        
+        //l10n.language.code = "e";
+        //console.log("In tutorial lang: "+language);
+        //console.log("In tutorial lang: "+l10n.language.code);
+		var steps = [
+			{
```
Comment (llaske) [inline activities/Fototoon.activity/lib/tutorial.js]: Try to be consistent about indentation in the file: should be TAB or white spaces but not both mixed

Comment (justnoone00) [reply activities/Fototoon.activity/lib/tutorial.js]: Fixed indentation.

**Comment by justnoone00** [comment]
Fixed issue and done all requested changes.

**Comment by llaske** [comment]
Sorry, your tutorial.js still mix different type of indentation TAB and white spaces. 
Should be always the same in a same file.

**Comment by justnoone00** [comment]
Sorry for confusion, finally improved tutorial.js indentation.

**Pull request merged**